test/compilable/extra-files/ddoc17697.html

```diff
@@ -1,484 +1,506 @@
-
```

Comment:
> So what difference does this line make if it's in a separate commit?

The question still makes sense despite the above, so to answer that - because it makes it actually possible to see only the lines that actually changed.